src/gui/speedlimitdlg.cpp

```diff
@@ -104,7 +104,7 @@ void SpeedLimitDialog::setupDialog(long max_slider, long val) const
     if (val < 0)
         val = 0;
     if (max_slider <= 0)
-        max_slider = 1000;
+        max_slider = 10000;
```

Comment:
what about it? the max is controlled only here. IMO the slider max in bandwidth_limit.ui should get reset to default value.